package.json

```diff
@@ -31,10 +29,7 @@
     "ember-cli-inject-live-reload": "^1.4.1",
     "ember-cli-qunit": "^4.0.0",
     "ember-cli-shims": "^1.1.0",
```

Comment:
can you remind me when we need ember-cli-shims?

Reply:
All subdependencies need to have `ember-cli-babel@^6.6.0`. I just checked and we could throw it out, but at the moment Ember CLI will issue an unnecessary warning when you do it so I'd avoid that for now.

Reply:
> at the moment Ember CLI will issue an unnecessary warning when you do it so I'd avoid that for now

FWIW, this is fixed in latest beta branch (so as of ember-cli@2.17 removing ember-cli-shims will not issue a warning if all addons have at least ember-cli-babel@6.6.2).

Reply:
oh awesome, must have missed that :)